lib/ui/screens/artifact/artifact_details/widgets/_info_column.dart

```diff
@@ -18,7 +18,7 @@ class _InfoColumn extends StatelessWidget {
                 Text(
                   data.culture.toUpperCase(),
                   style: $styles.text.titleFont.copyWith(color: $styles.colors.accent1),
-                ).animate().fade(delay: 150.ms, duration: 600.ms),
+                ).animate().fade(delay: $styles.customTime.delay(150), duration: $styles.times.med),
```

Comment:
I don't love this, but am struggling to think of a good alternatives :D, in the meantime can we change this syntax to `$styles.times.ms(150)`? 

I don't think we need to differentiate between delay/animation. And also, what do we use `always` for?